accounts/management/commands/update_majors.py

```diff
@@ -0,0 +1,73 @@
+import requests
+from bs4 import BeautifulSoup
+from django.core.management.base import BaseCommand
+
+from accounts.models import Major
+
+
+def contains_filters(listed_filters, desired_filters=set(), excluded_filters=set()):
+    # ensure no excluded filters appear
+    for curr_filter in excluded_filters:
+        if curr_filter in listed_filters:
+            return False
+
+    # ensure at least one desired filter appears
+    for curr_filter in desired_filters:
+        if curr_filter in listed_filters:
+            return True
+
+    return False
+
+
+class Command(BaseCommand):
```

Comment:
Can you move the logic for these commands into a different file (like `scraping.py` or something. I think it makes things a bit clearer and testing easier. We do something similar in ohq ([command](https://github.com/pennlabs/office-hours-queue/blob/master/backend/ohq/queues.py) and [logic](https://github.com/pennlabs/office-hours-queue/blob/master/backend/ohq/queues.py)).

Reply:
fixed